src/govuk/components/details/details.js

```diff
@@ -128,27 +90,54 @@ Details.prototype.setAttributes = function () {
   $summary.setAttribute('aria-expanded', (expanded ? 'false' : 'true'))
   $content.setAttribute('aria-hidden', (hidden ? 'false' : 'true'))
 
-  if (!NATIVE_DETAILS) {
-    $content.style.display = (expanded ? 'none' : '')
+  $content.style.display = (expanded ? 'none' : '')
 
-    var hasOpenAttr = $module.getAttribute('open') !== null
-    if (!hasOpenAttr) {
-      $module.setAttribute('open', 'open')
-    } else {
-      $module.removeAttribute('open')
-    }
+  var hasOpenAttr = $module.getAttribute('open') !== null
```

Comment:
I do wonder what the benefit is of adding the `open` attribute to a non-native `details` element. It seems a bit strange, but I guess it might help some AT announce the state? 🤷‍♂

Reply:
I suppose if people have styled using the 'open' attribute?